packages/app-mobile/components/screens/Notes.tsx

```diff
@@ -223,27 +224,43 @@ class NotesScreenComponent extends BaseScreenComponent<any> {
 		let buttonFolderId = this.props.selectedFolderId !== Folder.conflictFolderId() ? this.props.selectedFolderId : null;
 		if (!buttonFolderId) buttonFolderId = this.props.activeFolderId;
 
-		const addFolderNoteButtons = !!buttonFolderId;
+		const isAllNotes =
+			this.props.notesParentType === 'SmartFilter'
+			&& this.props.selectedSmartFilterId === ALL_NOTES_FILTER_ID;
+
+		// Usually, when showing all notes, activeFolderId/selectedFolderId is set to the last
+		// active folder.
+		// If the app starts showing all notes, activeFolderId/selectedFolderId are
+		// empty or null. As such, we need a special case to show the buttons:
+		const addFolderNoteButtons = !!buttonFolderId || isAllNotes;
 		const thisComp = this;
 
 		const makeActionButtonComp = () => {
+			const getTargetFolderId = async () => {
+				if (!buttonFolderId && isAllNotes) {
+					return (await Folder.defaultFolder()).id;
```

Comment:
I think you can use the activeFolderId setting here, which would be the last selected folder

Reply:
The source of the missing note button issue seems to be that `activeFolderId` is either `''` or `null` on startup when opening the "all notes" screen — in this case, there is no previous folder.